tests/core/test_utils.py

```diff
@@ -44,12 +59,12 @@ def test_float(self):
         self.assertEqual(native_value('3.14'), 3.14)
 
     def test_true_values(self):
-        true_strings = ['True', 'on', 'true', 'yes']
```

Comment:
Because they are not valid JSON values and we don't use it, so I'd like to change the behavior to stick with the JSON spec there.